src/overlays/actors/ovl_En_Dekunuts/z_en_dekunuts.c

```diff
@@ -15,20 +16,32 @@ void EnDekunuts_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnDekunuts_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnDekunuts_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_060001C4;
-extern UNK_TYPE D_06000368;
-extern UNK_TYPE D_060004D8;
-extern UNK_TYPE D_060006B0;
-extern UNK_TYPE D_060008C4;
-extern UNK_TYPE D_06000AF0;
-extern UNK_TYPE D_06000D1C;
-extern UNK_TYPE D_06000E6C;
-extern UNK_TYPE D_06001024;
-extern UNK_TYPE D_06002298;
-extern UNK_TYPE D_06003268;
-extern UNK_TYPE D_06003650;
+void EnDekunuts_SetupWait(EnDekunuts* this);
+void EnDekunuts_Wait(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_LookAround(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_Stand(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_ThrowNut(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_Burrow(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_BeginRun(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_Run(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_Gasp(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_BeDamaged(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_BeStunned(EnDekunuts* this, GlobalContext* globalCtx);
+void EnDekunuts_Die(EnDekunuts* this, GlobalContext* globalCtx);
+
+extern AnimationHeader gDekuNutsSpitAnim;
```

Comment:
We dont need to `extern ` these anymore since they are externed in the object file